odata-json-format/10 Media Entity.md

```diff
@@ -33,19 +33,24 @@ An individual property or operation response is represented as a JSON
 object.
 
 A single-valued property or operation response that has the
-`null` value does not have a representation; see
-[OData-Protocol](#ODataProtocol).
+`null` value does not have a representation provided that it carries no
```

Comment:
What does this mean when this section is applied recursively, while representing the properties of an entity?

Section 6 says "each structural property is represented as a name/value pair", and I understand that the value is the representation of the property. But now this section here says that the property "has no representation".

Reply:
This section describes the full response body of an individual property or operation response. It is not intended to be applied recursively.

Reply:
Similar to collection responses: they are also not intended for recursive application, otherwise instead of `"coll":[...]` we would see `"coll":{"value":[...]}`